src/pages/Home/Home.styled.jsx

```diff
@@ -1,133 +1,114 @@
+
+  @media (max-width: 992px) {
+    position: absolute;
+    top: 27%;
+    column-gap: 0;
   }
+`
+export const HalfSection = styled.div`
+  flex-basis: 45%;
+  align-self: center;
 
-  &.images-section {
+  @media (max-width: 992px) {
+    margin: 0 auto;
     display: flex;
-    justify-content: center;
+    flex-direction: column;
+    justify-content: space-between;
+    height: 85vh;
+    flex-basis: 80%;
   }
 `
-export const Logo = styled.img`
-  display: none;
-  max-width: 30%;
-  @media (min-width: 768px) {
-    display: block;
-    position: absolute;
-    top: 4%;
-    left: 4%;
-    height: 32px;
+export const ManImage = styled.img`
+  object-fit: contain;
+  max-width: 291px;
+  height: 68vh;
+
+  @media (max-width: 992px) {
+    width: 35%;
+    position: relative;
+    top: -100px;
   }
 `
-export const ManImage = styled.img`
-  display: block;
-  width: 70%;
-  position: absolute;
-  top: 25%;
-  left: 50%;
-  transform: translateX(-50%);
+export const DeliveryManCar = styled.img`
+  max-width: 210px;
+  position: relative;
+  bottom: 30px;
   object-fit: contain;
-  height: 60vh;
-  @media (min-width: 768px) {
-    transform: none;
-    position: static;
-    align-self: center;
-    max-width: 500px;
-    height: 68vh;
+
+  @media (max-width: 992px) {
+    width: 50%;
   }
 `
 export const TextContainer = styled.div`
-  max-width: 100%;
-  margin-top: 3rem;
-  @media (min-width: 768px) {
-    margin-top: 0px;
-    max-width: 70%;
+  margin: 0 auto;
+  max-width: 512px;
+  color: var(--headertext);
+  margin-bottom: 126px;
+
+  @media (max-width: 992px) {
+    margin: 0;
   }
 `
-export const Text = styled(OriginalText)`
-  ${[text.textCenter]}
-  ${props => props.title && text['7xl']}
-`
-export const DisplayText = styled(Text)`
-  font-size: 3rem;
-  text-align: left;
-  margin: 0px;
+export const DisplayText = styled.h1`
+  letter-spacing: -2px;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 700;
+  font-size: 69px;
   text-align: left;
-  @media (min-width: 768px) {
-    font-size: 5rem;
-    text-align: left;
-    line-height: 1;
-  }
-  @media (min-width: 1025px) {
-    font-size: 6.5rem;
+  margin: 0;
+
+  @media (max-width: 992px) {
+    font-size: 43px;
   }
 `
-export const SubtitleText = styled(Text)`
+export const SubtitleText = styled.p`
+  font-family: 'Poppins', sans-serif;
+  font-weight: 500;
+  font-size: 30px;
   margin-top: 3px;
-  font-size: 1rem;
   text-align: left;
-  @media (min-width: 1025px) {
-    font-size: 2rem;
-    line-height: 1;
-  }
-  @media (min-width: 768px) {
-    font-size: 1.5rem;
+
+  @media (max-width: 992px) {
+    font-size: 17px;
   }
 `
 export const Button = styled(Link)`
-  background: var(--primary);
+  max-width: 350px;
+  padding: 20px 0;
+  background: var(--light-blue-11);
   border-radius: 20px;
   border: none;
   color: var(--white);
-  width: 70%;
-  max-height: 40px;
   display: block;
-  margin: 10px auto 0px;
+  margin: auto;
+  margin-top: 56px;
```

Comment:
You can use too: 

```suggestion
  margin: 56px auto 0;
```

![imagen](https://user-images.githubusercontent.com/66505715/158837742-8ab3914f-0b52-4777-aefa-ac48b58befec.png)